src/main/java/uk/gov/hmcts/reform/ccd/documentam/service/DocumentManagementService.java

```diff
@@ -21,8 +21,8 @@
 
     void patchDocumentMetadata(CaseDocumentsMetadata caseDocumentsMetadata);
 
-    String generateHashToken(final UUID documentId, final Document document, final String defaultCaseTypeId,
-                             final String defaultJurisdictionId);
+    String generateHashToken(final UUID documentId, final String caseId, final String caseTypeId,
```

Comment:
I guess case type id and jurisdiction id would need to swap orders.

Reply:
done